assets/js/sections/qso.js

```diff
@@ -1302,7 +1302,35 @@ function getContinentResult() {
 	});
 }
 
-// This function executes the call to the backend for fetching iota summary and inserted table below qso entry
+// This function executes the call to the backend for fetching DOK summary and inserts table below qso entry
+function getDokResult() {
+	satOrBand = $('#band').val();
+	if ($('#selectPropagation').val() == 'SAT') {
+		satOrBand = 'SAT';
+	}
+	$('#dok-summary').empty();
+	if ($('#darc_dok').val() === '') {
+		$('#dok-summary').append(lang_summary_warning_empty_iota);
```

Comment:
Probably dok here :)

Reply:
@phl0 yes, you are right. I missed that in the copy/paste. Fixed now. Thanks :)